client_plugin/drivers/utils/pluginDriver.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+//
```

Comment:
1. Package level comment should be above L15 .. that means before `package xxx`
2. file name should be corrected too 

Reference: "By convention, packages are given lower case, single-word names; there should be no need for underscores or mixedCaps."

Generally, file names follow the same convention as package names (https://golang.org/doc/effective_go.html#package-names)